test/integration/discover_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"testing" // should be imported to enable testing for GO modules
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft" // should be
+	// imported to enable the Blueprints helper modules for the test framework
+)
+
+func TestAll(t *testing.T) { // entry function for the test; can be named as Test*
```

Comment:
Where are we invoking the test in CI?
I suggest we add a new build stage for go tests before we phase out the others